### ISSUE TITLE
Failed to parse time string

### ISSUE THREAD

**Issue by kasimi** [phpbb-extensions/autogroups#138]
I got an error log where this happens a lot:

```
PHP Fatal error: Uncaught Exception: DateTime::__construct(): Failed to parse time string (99-0-9999) at position 6 (9): Unexpected character in ext/phpbb/autogroups/conditions/type/birthdays.php:114
```

**Comment by VSEphpbb** [comment]
Can you find out what that unexpected character is?

**Comment by kasimi** [comment]
It says in the error message. `$user_birthday` is `99-0-9999`

**Comment by VSEphpbb** [comment]
OK, so now how is it possible that you have a user with that as a birthday (a date which has invalid characters)?

**Comment by kasimi** [comment]
Good question. I hadn't checked if phpBB validates the date, turns out [it does](https://github.com/phpbb/phpbb/blob/release-3.2.3/phpBB/includes/ucp/ucp_profile.php#L334-L336). It must have been a rogue extension. I will fix the value manually.

Feel free to close this.

**Comment by VSEphpbb** [comment]
Fixed instead #140
